application/clicommands/CheckCommand.php

```diff
@@ -72,8 +72,16 @@ public function hostAction()
                 'cc.invalid_reason',
                 'c.subject',
                 'self_signed'   => 'COALESCE(ci.self_signed, c.self_signed)',
-                'c.valid_from',
-                'c.valid_to'
+                'valid_from'    => (new Select())
+                    ->from('x509_certificate_chain_link xccl')
+                    ->columns('MAX(xc.valid_from)')
+                    ->join('x509_certificate xc', 'xc.id = xccl.certificate_id')
+                    ->where('xccl.certificate_chain_id = cc.id'),
```

Comment:
Well, it didn't work for me. However, I modified the query a bit and got it.

### Before

```bash
root@a2f6d2aaab32:/# icingacli x509 check --ip 10.211.55.31 --port 443
OK - Icinga Web 2 expires in 14 days|'Icinga Web 2'=1241677s;345600;172800;0;1296000
```

### After

```bash
root@a2f6d2aaab32:/# icingacli x509 check --ip 10.211.55.31 --port 443
OK - Icinga Web 2 expires in 4 days|'Icinga Web 2'=377696s;86400;0;0;424139
```

```suggestion
                    ->from('x509_certificate xc')
                    ->columns('MAX(xci.valid_from)')
                    ->join('x509_certificate xci', 'xci.issuer_hash = xc.issuer_hash')
                    ->where('xc.id = c.id'),
```